integration-tests/smoke/vrfv2_test.go

```diff
@@ -0,0 +1,187 @@
+package smoke_test
+
+//revive:disable:dot-imports
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+	uuid "github.com/satori/go.uuid"
+	"github.com/smartcontractkit/helmenv/environment"
+	"github.com/smartcontractkit/helmenv/tools"
+	"github.com/smartcontractkit/integrations-framework/actions"
+	"github.com/smartcontractkit/integrations-framework/client"
+	"github.com/smartcontractkit/integrations-framework/contracts"
+	"github.com/smartcontractkit/integrations-framework/contracts/ethereum"
+	"github.com/smartcontractkit/integrations-framework/utils"
+)
+
+var _ = Describe("VRFV2 suite @v2vrf", func() {
+	var (
+		err                error
+		nets               *client.Networks
+		cd                 contracts.ContractDeployer
+		consumer           contracts.VRFConsumerV2
+		coordinator        contracts.VRFCoordinatorV2
+		encodedProvingKeys = make([][2]*big.Int, 0)
+		lt                 contracts.LinkToken
+		cls                []client.Chainlink
+		e                  *environment.Environment
+		vrfKey             *client.VRFKey
+		job                *client.Job
+		// used both as a feed and a fallback value
+		linkEthFeedResponse = big.NewInt(1e18)
+	)
+
+	BeforeEach(func() {
+		By("Deploying the environment", func() {
+			e, err = environment.DeployOrLoadEnvironment(
+				environment.NewChainlinkConfig(
+					nil,
+					"",
+					// works only on perf Geth
+					environment.PerformanceGeth,
+				),
+				tools.ChartsRoot,
+			)
+			Expect(err).ShouldNot(HaveOccurred())
+			err = e.ConnectAll()
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+
+		By("Connecting to launched resources", func() {
+			networkRegistry := client.NewDefaultNetworkRegistry()
+			nets, err = networkRegistry.GetNetworks(e)
+			Expect(err).ShouldNot(HaveOccurred())
+			cd, err = contracts.NewContractDeployer(nets.Default)
+			Expect(err).ShouldNot(HaveOccurred())
+			cls, err = client.ConnectChainlinkNodes(e)
+			Expect(err).ShouldNot(HaveOccurred())
+			nets.Default.ParallelTransactions(true)
+		})
+		By("Funding Chainlink nodes", func() {
+			err = actions.FundChainlinkNodes(cls, nets.Default, big.NewFloat(3))
+			Expect(err).ShouldNot(HaveOccurred())
+			// Edge case where simulated networks need some funds at the 0x0 address in order for keeper reads to work
```

Comment:
Is this needed for a VRFv2 test?

Reply:
Yes, `0x0` must be funded otherwise `simulate` part of a pipeline will fail

Reply:
Just update the comment then to reflect that

Reply:
I'd also add this as a constant so we're not copying this string in tests